src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -335,7 +335,7 @@ private void processFiles(RootModule rootModule, final SeverityLevelCounter warn
         log("To locate the files took " + (endTime - startTime) + TIME_SUFFIX,
             Project.MSG_VERBOSE);
 
-        log("Running Checkstyle "
+        log("Running Checkstyle"
```

Comment:
`log("Running Checkstyle of version '"`
Will be better to wrap version with `'`

Reply:
yes my friend, but please respect SOC and `start finishing and stop starting`; merging this and then doing next thing in next thing (PR). Giving each dedicated concern.


We tend to go from hero to zero and getting noting done. Its breaking change and makes all harder. I have refactored again because of the interconnection. 

In Order to go fast we need to go right. ― Robert C. Martin

https://www.goodreads.com/quotes/9701819-the-only-way-to-go-fast-is-to-go-well

Reply:
https://github.com/checkstyle/checkstyle/pull/17043

Reply:
please send my best regards to Robert C. Martin.
and proceed with 
`log("Running Checkstyle of version "`
or
`log("Running Checkstyle of version '"` (wrapping with `'`).


I even think variant without `'` is better

Reply:
Yes can do. This `of version` is kind of overhead, as its kind of clear.

Running Checkstyle 2.3
Running Checkstyle of version 2.3
Running Checkstyle v2.3

Reply:
Starting MyApplication using Java 17.0.15

 Starting MyApplication using Java of Version 17.0.15

its nice to be clear and give infos but too much is overhead. 

` Starting SampleApplication v0.1.0 `

https://docs.spring.io/spring-boot/docs/2.1.x/reference/html/boot-features-spring-application.html

<img width="1216" alt="image" src="https://github.com/user-attachments/assets/e597a6f4-5df4-455f-88a1-392878340149" />

https://docs.spring.io/spring-boot/reference/features/logging.html

Reply:
Ok, in this case we don't need this update.
All is already clear.

Reply:
Lets fix typo then lets continue. The version is handed over 4 different layers for no reason its a static value.

Lets fix whitespace then lets see. Thx.